product/roundhouse.databases.mysql/MySqlDatabase.cs

```diff
@@ -109,7 +111,15 @@ private static string build_connection_string(string server_name, string databas
         public override string create_database_script()
         {
             return string.Format(
-                @"CREATE DATABASE IF NOT EXISTS `{0}`;",
+                @"SET @Created = 1;
+
+                SELECT 0 FROM INFORMATION_SCHEMA.SCHEMATA
+                WHERE SCHEMA_NAME='{0}' INTO @Created;
+
+                CREATE DATABASE IF NOT EXISTS `{0}`
+                CHARACTER SET utf8 COLLATE utf8_unicode_ci;

```

Comment:
It looks like this is not the default collation for MySQL. Am I wrong on that?

I'd like to address collation separately and decide if changing the default behavior is a good thing.